src/lib/components/error-pages/ErrorPage404.component.js

```diff
@@ -0,0 +1,75 @@
+//@flow
+import React from "react";
+import styled from "styled-components";
+import * as defaultTheme from "../../style/theme";
+import { getTheme, getThemePropSelector } from '../../utils';
+import Button from '../button/Button.component';
+
+const ErrorPageContainer = styled.div`
+  background-color: ${getThemePropSelector("primaryDark1")};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+`;
+
+const Title = styled.h2`
+  color: ${props => getTheme(props).textPrimary};
+  margin: 0;
+  margin-left: ${defaultTheme.padding.large};
+`;
+
+const Description = styled.div`
+  color: ${props => getTheme(props).textPrimary};
+  font-size: ${defaultTheme.fontSize.larger};
+  text-align: center;
+  padding: ${defaultTheme.padding.larger};
+`;
+
+const DescriptionContent = styled.p`
+  margin: 0;
+  padding: ${defaultTheme.padding.smaller};
+`;
+
+const Row = styled.div`
+  display: flex;
+  align-items: baseline;
+`;
+
+const WarningIcon = styled.i`
+  color: ${defaultTheme.yellow};
+`;
+
+type Props = {
+  btnLink?: string,
+};
+
+function ErrorPage404({
+  btnLink = "/",
+  ...rest
+}: Props) {
+  return (
+    <ErrorPageContainer className="sc-error-page404" { ...rest }>
+      <Row>
+        <WarningIcon className="fas fa-exclamation-triangle fa-2x"/>
+        <Title>Error: this page does not exist</Title>
+      </Row>
+      <Row>
+        <Description>
+          <DescriptionContent>It might be the wrong address or the page has moved.</DescriptionContent>
```

Comment:
Hi @ChengYanJin, for the translation i suggest:
It is okay for you?

404:
Error: this page does not exist -> Erreur: cette page n'existe pas
It might be the wrong address or the page has moved. -> Il se peut que ce soit une mauvaise adresse ou que la page ait été déplacée.
You might check the url, or contact your admin if the error remains -> Vous pouvez vérifier l'url, ou contacter votre administrateur si l'erreur persiste.

505:
Unexpected Error -> Erreur inattendue
An error occured and your request cannot be completed. -> Une erreur s'est produite et votre demande ne peut être complétée.
You might check the url, or contact your admin if the error remains. -> Vous pouvez vérifier l'url, ou contacter votre administrateur si l'erreur persiste.
You may also contact support to report this issue. -> Vous pouvez également contacter le support pour signaler ce problème.

Reply:
Hi @theocerutti 
The translations look good for us. Thanks for the proposal!